internal/discover/symlinks_test.go

```diff
@@ -308,10 +308,12 @@ func TestWithWithDriverDotSoSymlinks(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.description, func(t *testing.T) {
+			hookCreator := NewHookCreator("/path/to/nvidia-cdi-hook")
```

Comment:
This can be defined outside of the `for` loop.

Reply:
done